Source/WTF/wtf/PlatformHave.h

```diff
@@ -1138,6 +1138,7 @@
 
 #if HAVE(WEBGPU_IMPLEMENTATION) && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 130000) || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 160000))
 #define HAVE_TIER2_ARGUMENT_BUFFERS 1
+#define HAVE_METAL_BUFFER_BINDING_REFLECTION 1
```

Comment:
I guess this is all throwaway code anyway, but I think reflection is available way further back than this macro indicates.

Reply:
Yes, but the reflection APIs used are new to iOS 16 / macOS 13, the older ones are deprecated